src/llm/apis/openai_completions.hpp

```diff
@@ -140,6 +146,11 @@ struct OpenAIChatCompletionsRequest {
 
         if (logprobschat || logprobs > 0)
             config.logprobs = 1;
+        // Speculative decoding specific
+        if (numAssistantTokens.has_value())
+            config.num_assistant_tokens = numAssistantTokens.value();
+        if (assistantConfidenceThreshold.has_value())
```

Comment:
I dont see a place where setting both parameters (which are exclusive) fails the request, do we have test for that?

Reply:
https://github.com/openvinotoolkit/model_server/pull/2777/files#diff-dc1e3d8f1c59d392baedaaa031edd2ef7bbb062fa25cd68fe99b00011f9f770bR492

Reply:
We have only positive tests now

Reply:
added negative tests